docs/helpful-guides/how-to-use-recaptcha-widget.md

```diff
@@ -30,34 +30,47 @@ Once you have the API site key you will also be displayed a secret key. You have
 Override the Form 
 -----------------
 
-For the sake of the example, we are going to override the `Da\User\Form\RecoveryForm` class: 
+For the sake of the example, we are going to override the `Da\User\Form\RecoveryForm` class. Create a new file `RecoveryForm`
+add it to @app/models/Forms/ and put the following in it:
 
-```php 
-namespace app\forms;
+```
+<?php 
+namespace app\models\Forms;
+
+use Da\User\Form\RecoveryForm as BaseForm;
+
+class RecoveryForm extends BaseForm {
 
-class RecoveryForm extends Da\User\Form\RecoveryForm {
-    
     public $captcha;
-    
+
     public function rules() {
-    
+
         $rules = parent::rules();
-        
+
         $rules[] = [['captcha'], 'required'];
         $rules[] = [['captcha'], 'Da\User\Validator\ReCaptchaValidator'];
-        
+
         return $rules;
     }
+    
+    public function scenarios()
+    {
+        return [
+            self::SCENARIO_REQUEST => ['email', 'captcha'],
+            self::SCENARIO_RESET => ['password'],
+        ];
+    }
 }
 
 ```
 
+
 Overriding the View
 -------------------
 
 Create a new file and name it `request.php` and add it in `@app/views/user/recovery`. Add the captcha widget to it: 
 
-```php 
+``` 
```

Comment:
Why did you remove the lang tag?

Reply:
It's on the next line as ````<?php````

Reply:
Does it work anyway? Because they're different things: the removed one is a Markdown syntax, while the one below is programming

Reply:
Github markdown extension I didn't know about.  Works either way.  It appears that the syntax coloring still happens.